libpoolprotocols/PoolClient.h

```diff
@@ -5,17 +5,48 @@
 
 #include <libethcore/Farm.h>
 #include <libethcore/Miner.h>
+#include <libpoolprotocols/PoolURI.h>
 
 using namespace std;
 
 namespace dev
 {
 	namespace eth
 	{
+		class PoolConnection
```

Comment:
We should think about the ability to return the IP-Address or something from the PoolClient back to manager using the PoolConnection object in the future.

Reply:
Indeed.

Reply:
Added accessor methods for IP address.